src/main/java/org/kaazing/netx/ws/internal/io/OutgoingSentinelExtension.java

```diff
@@ -288,4 +291,18 @@ else if (reasonLength > 123) {
         }
     }
 
+    private void createMask(byte[] maskBuf) {
+        int m = maskInt.incrementAndGet();
```

Comment:
Good point. I will change it to

``` java
int m = 1 + random.nextInt(Integer.MAX_VALUE);
```

We are moving away from SecureRandom.nextBytes() as it is creating garbage.